src/Bicep.Core/LanguageConstants.cs

```diff
@@ -70,6 +70,7 @@ public static class LanguageConstants
         public const string ResourceScopePropertyName = "scope";
         public const string ResourceParentPropertyName = "parent";
         public const string ResourceDependsOnPropertyName = "dependsOn";
+        public const string TypeNameString = "string";
```

Comment:
>TypeNameString [](start = 28, length = 14)

Would `StringTypeName` be better?

Reply:
I thought about this - I'm going less specific -> more specific left-to-right so that e.g. `TypeNameInteger`, `TypeNameString`, `TypeNameWhateverElse` are grouped alphabetically.

Reply:
Which is different from most of the constants in the class :P

---
In reply to: [598960954](https://github.com/Azure/bicep/pull/1950#discussion_r598960954) [](ancestors = 598960954)